FirebaseMLModelDownloader/Sources/RemoteModelInfo.swift

```diff
@@ -13,22 +13,17 @@
 // limitations under the License.
 
 import Foundation
-import FirebaseCore
 
 /// Model info object with details about not-yet downloaded model.
 struct RemoteModelInfo {
   /// Model name.
   let name: String
-
```

Comment:
Same comment here re: leaving the spaces, was this swift-format or manual?

Reply:
I'm not sure, but I added spaces between properties everywhere so it's consistent and readable.